src/app/sign-in/components/two-factor/two-factor.component.scss

```diff
@@ -6,6 +6,16 @@
   margin-top: 12px;
 }
 
+.error-message {
+  float: left;
```

Comment:
I believe this is going to give you problems on Arabian

Reply:
Test it please and if that's the case please create an overwrite for RTL like this 

https://github.com/ORCID/orcid-angular/blob/5b629161eab8f0427135bc0641f3f4553d0e86e3/src/app/home/pages/home/home.component.scss#L90-L92

This will apply for whenever you use explicit `right` or `left` values

Reply:
Ok